charts/dispatch/charts/image-manager/values.yaml

```diff
@@ -26,20 +26,21 @@ ingress:
   tls: {}
     # Secrets must be manually created in the namespace.
     # secretName: dispatch-tls
-resources: {}
+resources:
   # We usually recommend not to specify default resources and to leave this as a conscious
   # choice for the user. This also increases chances charts run on environments with little
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #  cpu: 100m
-  #  memory: 128Mi
-  #requests:
-  #  cpu: 100m
-  #  memory: 128Mi
+  limits:
```

Comment:
We might want to change the above comment block if we are going to be uncommenting these fields.